fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -1040,6 +1040,23 @@ struct ConsistencyCheckWorkload : TestWorkload {
 		return true;
 	}
 
+	// Run an empty commit through the system.
+	ACTOR static Future<Void> doEmptyCommit(Database cx) {
+		state Transaction tr(cx);
+		loop {
+			try {
+				wait(::success(tr.getReadVersion()));
+				tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
```

Comment:
I think priority system immediate would only have an effect if set prior to getting the read version

Reply:
Good point. Let me move the read version below that.